src/grid/componentGrid.js

```diff
@@ -10,19 +10,23 @@ const ComponentGridComponent = props => {
         headerClass,
         rowClickHandler,
         style,
+        useDefaultStyle,
         rowsContainerStyle,
         rowHeader,
         rowFooter
     } = props;
 
+    const setDefaultStyle = (typeof useDefaultStyle === 'undefined') ? true : useDefaultStyle;
```

Comment:
I think my brain just exploded LOL... is there a way we could write this to make it easier to reason about?

does this work:

const setDefaultStyle = useDefaultStyle || true;

?